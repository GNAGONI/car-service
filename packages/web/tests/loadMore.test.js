import React from 'react'
import renderer from 'react-test-renderer'
import LoadMore from 'components/loadMore'

it('Renders correctly', () => {
  const tree = renderer.create(<LoadMore />).toJSON()
  expect(tree).toMatchSnapshot()
})
