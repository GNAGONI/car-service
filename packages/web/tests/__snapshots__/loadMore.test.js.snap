// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="load-more"
>
  <div
    className="load-more__content mb-5"
  />
  <div
    className="load-more__footer text-center"
  >
    <button
      className="btn btn-new btn-success"
      onClick={[Function]}
    >
      Load more
    </button>
  </div>
</div>
`;
